lib/mix/tasks/configure_esbuild.ex

```diff
@@ -10,7 +10,7 @@ defmodule Mix.Tasks.LiveSvelte.ConfigureEsbuild do
 
     Mix.Project.deps_paths(depth: 1)
     |> Map.fetch!(:live_svelte)
-    |> Path.join("assets/copy/**/*{.js}")
+    |> Path.join("assets/copy/**/*.{js,json}")
     |> Path.wildcard()
     |> Enum.each(fn file ->
```

Comment:
I think it does only this when I logged it to the terminal.
```suggestion
    |> Enum.each(fn full_path ->
      [_beginning, relative_path] = String.split(full_path, "assets", parts: 2)

      Mix.Generator.copy_file(full_path, "assets" <> relative_path)
    end)

```

Reply:
That's way better, thank you! Small adjustment, on line 16 "assets" -> "copy"

Fixed here: https://github.com/woutdp/live_svelte/pull/38/commits/fe0a51a0a827d969c3d13e21e6b5fb716bcf7292